python/paddle/v2/framework/tests/test_layers.py

```diff
@@ -20,7 +20,7 @@ def test_fit_a_line(self):
         avg_cost = layers.mean(x=cost, program=program)
         self.assertIsNotNone(avg_cost)
         program.append_backward(avg_cost)
-        print str(program)
+        # print str(program)
```

Comment:
Do not change the previous unit tests.

You can select the test method in runtime, by

```bash
python -m unittest test_layers.TestBook.test_fit_a_line
```

Reply:
Done